src/scss/components/_helpers.scss

```diff
@@ -27,6 +28,75 @@
   position: relative;
 }
 
+.static,
+%static {
+  position: static;
+}
+
+.sticky,
+%sticky {
+  position: sticky;
+}
+
+// Display
+
+.display-flex,
```

Comment:
i think we should use BEM here. main reason being we want to separate that this is a display helper from what the display helper does. i.e.

```
.display--inline-block {
}
```

vs

```
.display-inline-block {}
```

What are your thoughts?

Reply:
I'll add, ply is currently using BEM some places, some places it isn't. I think it should use BEM throughout.

Reply:
Good point. I honestly did not consider BEM in this approach and was just thinking: `{attribute}-{property}` 

For all the border, padding, and margin stuff, it follow this same, non-BEM pattern. I do think that BEM could address my original concern about creating additional (future) classes to discern from `display: inline-table` and `display: inline table;` i.e. we can do: `display--inline-table` and `display--inline_table` ?

For these display ones, since they are new, let's just go ahead and use BEM syntax. I'll make the update and push changes in a moment.